beacon_chain/state/helpers.py

```diff
@@ -131,3 +134,26 @@ def get_new_shuffling(seed,
             committee=indices
         ) for j, indices in enumerate(shard_indices)])
     return o
+
+
+def get_indices_for_slot(
+        crystallized_state,
+        slot,
+        config=DEFAULT_CONFIG):
+    cycle_length = config['cycle_length']
+
+    ifh_start = crystallized_state.last_state_recalc - cycle_length
+    assert ifh_start <= slot < ifh_start + cycle_length * 2
```

Comment:
What does `ifh` stands for?

Reply:
Oh, `indices_for_height`, but it was renamed to `indices_for_slot`. I will update it. Thanks.